plugin-flex-ts-template-v2/src/feature-library/custom-transfer-directory/flex-hooks/notifications/CustomTransferDirectory.ts

```diff
@@ -0,0 +1,21 @@
+import * as Flex from '@twilio/flex-ui';
+
+import { StringTemplates } from '../strings/CustomTransferDirectory';
+
+export enum CustomTransferDirectoryNotification {
+  FailedLoadingInsightsClient = 'FailedLoadingInsightsClient',
+  FailedLoadingInsightsData = 'FailedLoadingInsightsData',
+}
+
+export const notificationHook = (flex: typeof Flex, _manager: Flex.Manager) => [
+  {
+    id: CustomTransferDirectoryNotification.FailedLoadingInsightsClient,
+    content: StringTemplates.FailedToLoadInsightsClient,
+    type: flex.NotificationType.warning,
+  },
+  {
+    id: CustomTransferDirectoryNotification.FailedLoadingInsightsData,
+    content: StringTemplates.FailedToLoadInsightsData,
+    type: flex.NotificationType.warning,
+  },
```

Comment:
Given that these are warnings and not critical, a timeout of 3-5 seconds might be nice here.